binder/binderfs.c

```diff
@@ -503,7 +511,11 @@ static struct inode *binderfs_make_inode(struct super_block *sb, int mode)
 	if (ret) {
 		ret->i_ino = iunique(sb, BINDERFS_MAX_MINOR + INODE_OFFSET);
 		ret->i_mode = mode;
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(6,6,0))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(6,7,0))
+		struct timespec64 now = inode_set_ctime_current(ret);
```

Comment:
Just a small suggestion: could [simple_inode_init_ts](https://elixir.bootlin.com/linux/latest/C/ident/simple_inode_init_ts) be used here and in the 2 other cases above/below?

Reply:
ok, and thank for your suggestion